notes/prepare-for-1.15.0.md

```diff
@@ -0,0 +1,8 @@
+# Prepare for Azure Management Libraries for Java 1.15.0 #
+
+Steps to migrate code that uses Azure Management Libraries for Java from 1.14 to 1.15 ...
+
+> If this note missed any breaking changes, please open a pull request.
+
+
+V1.15 is backwards compatible with V1.14 in the APIs intended for public use that reached the general availability (stable) stage in V1.x.
```

Comment:
There are couple breaking changes in the AKS space:
- withVirtualMachineCount() renamed to withAgentPoolVirtualMachineCount()
- removed withKeyVaultSecret() Fluent method (service does not expose it)
- reordered the defineAgentPool() Fluent to start with withVirtualMachineSize()

Reply:
Can you add these to https://github.com/Azure/azure-libraries-for-java/blob/morereadme115/notes/prepare-for-1.15.0.md? Thanks Milis!